exercises/pig-latin/src/example.c

```diff
@@ -0,0 +1,121 @@
+#include "pig_latin.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+static bool is_vowel(const char a)
```

Comment:
maybe use `letter` instead of `a` as the parameter name - C does prefer brevity and single letter parameter names are used in the standard library a lot, but A) the standard library is nearly _too_ brief (they are looking at efficiency of code size , as applications programmers we don't have that concern at this level) B) `a` is a confusing name when talking about vowels (we have to say ` a == 'a'` which works but can be confusing).

The main idea when deciding such things is to think what would be the least mental work for someone reading the code to understand when the don't already know what's going on.

Reply:
I suppose it is a matter of taste, because I personally find it easier for my eyes when functions/names are not too verbose. Anyway, I changed this to your suggestion.

Reply:
I'm fine with the short variable name if you use an idiomatic `c` (for `char`).